app/Filament/Admin/Resources/ServerResource/Pages/EditServer.php

```diff
@@ -145,6 +146,14 @@ public function form(Form $form): Form
                                     ->label(trans('admin/server.description'))
                                     ->columnSpanFull(),
 
+                                TextInput::make('avatar_url')
+                                    ->label('Server Avatar')
+                                    ->columnSpanFull()
+                                    ->live(debounce: 500)
+                                    ->url()
+                                    ->formatStateUsing(fn ($state) => $state ?? (new UiAvatarsProvider())->get($this->getRecord()))
+                                    ->prefix(fn ($state) => new HtmlString("<img src='$state' width='24vh' height='24vh' class='fi-avatar object-cover object-center rounded-md h-8 w-8 fi-tenant-avatar shrink-0'/>")),
```

Comment:
Use [`<x-filament::avatar>`](https://filamentphp.com/docs/3.x/support/blade-components/avatar) instead of `<img>`.